ConsoleApp1/Library.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Library
+    {
+        public ReaderDatabase readerCards;
+        public Book[] books;
+        public Library(ReaderDatabase readerCards, Book[] books)
+        {
+            this.readerCards = readerCards;
+            this.books = books;
+        }
+        public Library()
+        {
+            this.readerCards = new ReaderDatabase();
+            this.books = new Book[1];
+        }
+        public void DisplayBooks()
+        {
+            for (int i = 0; i < books.Length - 1; i++)
+            {
+                books[i].DisplayBookInfo();
+            }
+        }
+        public void AddBook(Book newBook)
+        {
+            books[^1] = newBook;
+            Array.Resize(ref books, books.Length + 1);
+        }
+        public int SearchBook(string title)
```

Comment:
Use pattern` bool TrySearchBook(string title, out Book book) `

Reply:
public bool TrySearchBook(string title, out int index)

I changed it like that, 'cause I need an index of array, not the actual book.